src/v0/destinations/ga4/transform.js

```diff
@@ -129,114 +109,49 @@ const responseBuilder = (message, { Config }) => {
     payload.name = evConfigEvent;
     payload.params = constructPayload(message, mappingConfig[name]);
 
-    if (item) {
-      // item
-      payload.params.items = getItem(message, item === 'YES');
-    } else if (itemList) {
-      // itemList
-      payload.params.items = getItemList(message, itemList === 'YES');
-    }
-
-    // for select_item and view_item event we take custom properties from properties
-    // excluding items/product properties
-    if (payload.name === 'select_item' || payload.name === 'view_item') {
-      // exclude event properties
-      let ITEM_EXCLUSION_LIST = getGA4ExclusionList(mappingConfig[name]);
-      // exclude items/product properties
-      ITEM_EXCLUSION_LIST = ITEM_EXCLUSION_LIST.concat(
-        getGA4ExclusionList(mappingConfig[ConfigCategory.ITEM.name]),
-      );
+    if (!USER_IDENTIFIERS_EVENTS.includes(payload.name)) {
```

Comment:
Where are we handling the `login`, `sign_up`, `generate_lead` event?

Reply:
from mapping config

`payload.params = constructPayload(message, mappingConfig[name]);`

all 3 event has mapping config